changelog

```diff
@@ -2,6 +2,7 @@ MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-andr
 
 vNext
 ----------
+- [PATCH] Fix authority data being persisted across different SDK instances (#2151)
```

Comment:
can we update this to reflect that we fixed it only in native auth flows?

Reply:
Done